src/app/src/main/kotlin/org/imperial/mrc/hint/exceptions/AdrException.kt

```diff
@@ -2,4 +2,8 @@ package org.imperial.mrc.hint.exceptions
 
 import org.springframework.http.HttpStatus
 
-class AdrException(key: String, status: HttpStatus, url: String) : HintException(key, status, url)
+class AdrException(
+    key: String,
+    status: HttpStatus,
+    url: Array<String> = arrayOf("")
+) : HintException(key, status, url)
```

Comment:
```suggestion
class AdrException(
    key: String,
    status: HttpStatus,
    url: String
) : HintException(key, status, arrayOf(url))
```
No need for url to be an array I don't think. And does it require a default? Shouldn't url always be provided?